### DISCUSSION TITLE
Linkedin Feature

### DISCUSSION THREAD

**Discussion by vantuilpinheiro** [PaulMcInnis/JobFunnel#159] [Q&A]
I've found a video talking about this awesome tool!
But now I have a doubt: Does it work with Linkedin? I haven't found any reference here yet.

**Comment by PaulMcInnis** [comment]
it currently does not, feel free to propose a feature PR

**Comment by PaulMcInnis** [comment]
OK I have linkedin working, coming very soon in v4.0 🫡
